js/views/ListingCard.js

```diff
@@ -108,55 +107,63 @@ export default class extends baseVw {
   onClickEdit(e) {
     app.loadingModal.open();
 
-    const fullListingFetch = this.fullListing.fetch()
-      .done(() => {
-        if (fullListingFetch.statusText === 'abort' || this.isRemoved()) return;
+    this.fetchFullListing()
+      .done(xhr => {
+        if (xhr.statusText === 'abort' || this.isRemoved()) return;
```

Comment:
If you add the .always callback first, couldn't it handle the checking for abort and isRemoved?

Reply:
Even if you opt out early out of the always() callback, that will have no bearing on the done() or fail() callbacks, which will still fire.

Am I misunderstanding you?

Reply:
No, you're right, I was misthinking thing.